### ISSUE TITLE
Bug: Image size limit

### ISSUE THREAD

**Issue by foomoon** [tomayac/esm-potrace-wasm#8]
This library produces a "RangeError: offset is out of bounds" when loading LARGE images.  I have not isolated the break point, but for reference [this image](https://images.pexels.com/photos/1054666/pexels-photo-1054666.jpeg) will definitely cause the problem condition and I'm sure I could find examples where it occurs on images half that size.   Changing input options does not fix the problem.  (Using safari and chrome on mac)

**Comment by tomayac** [comment]
Yes, I have also noticed this. I wonder if @sylmroz who added direct color support has an idea how to fix this?

**Comment by damianoCospex** [comment]
Hello. Are there any solutions to this bug? I am having the same issue

**Comment by damiano216** [comment]
Hello, is there a solution to this bug? I am having the same issue

**Comment by tomayac** [comment]
Unfortunately not yet. If you have the cycles to investigate, this was introduced in https://github.com/tomayac/esm-potrace-wasm/pull/5/files, which added direct color support.
